tests/WordTest.php

```diff
@@ -0,0 +1,29 @@
+<?php
+require('../Word.php');
+use PHPUnit\Framework\TestCase;
+
+class WordTest extends TestCase {
+
+    public function test_success_createWordCard() 
+    {
+        $name = 'beer';
+        $definition = 'a lovely drink';
+        $synonyms = 'ale, pint, lager';
+        $sentence = 'I love beer';
+
+        $testWord = new Word($name, $definition, $synonyms, $sentence);
+
+        $expectedResult = 
+        "<section class='wordContainer'>
+            <div class='item'>
+                <h2 class='dictionaryWord'>beer</h2>
+                <p class='stat'><span>Definition</span>:  a lovely drink</p>
+                <p class='stat'><span>Synonyms</span>:  ale, pint, lager</p>
+                <p class='stat'><span>Example Sentence</span>:  I love beer</p>
```

Comment:
i see you have used a span as i asked, which is better, but they should also have a class, so that if ever in future another span is added to the page, it doesn't automatically get the same styling as these particular ones